Makefile

```diff
@@ -1,8 +1,16 @@
 CC=gcc
 CFLAGS=-O3 -fopenmp -Wall -Wextra -lX11
+PROG=i3lock-fancy-rapid
+BINDIR=/usr/bin
```

Comment:
I would prefer the default BINDIR to be `/usr/local/bin`.
Also please make use of the GNU standard [DESTDIR](https://www.gnu.org/prep/standards/html_node/DESTDIR.html).

Reply:
Yep I don't usually use MakeFiles, I'll change it